src/kdc/do_as_req.c

```diff
@@ -841,6 +841,8 @@ prepare_error_as(struct kdc_request_state *rstate, krb5_kdc_req *request,
     kdc_realm_t *kdc_active_realm = rstate->realm_data;
     size_t count;
 
+    memset(&errpkt, 0, sizeof(krb5_error));
```

Comment:
Should I replace all memset calls by an initialization of errpkt.magic = KV5M_ERROR? This would look a lot nicer to me.